lib/lightning_web/live/project_live/settings.ex

```diff
@@ -53,13 +70,148 @@ defmodule LightningWeb.ProjectLive.Settings do
       |> Projects.change_project(project_params)
       |> Map.put(:action, :validate)
 
-    {:noreply, assign(socket, :changeset, changeset)}
+    {:noreply, assign(socket, :project_changeset, changeset)}
   end
 
   def handle_event("save", %{"project" => project_params} = _params, socket) do
     save_project(socket, project_params)
   end
 
+  def handle_event(
+        "set_failure_alert",
+        %{
+          "project_user_id" => project_user_id,
+          "value" => value
+        } = _params,
+        socket
+      ) do
+    project_user = Projects.get_project_user!(project_user_id)
+
+    changeset =
+      {%{failure_alert: project_user.failure_alert}, %{failure_alert: :boolean}}
+      |> Ecto.Changeset.cast(%{failure_alert: value}, [:failure_alert])
+
+    case Ecto.Changeset.get_change(changeset, :failure_alert) do
+      x when not is_nil(x) ->
```

Comment:
Lets avoid using `x` and other unrelated letters for variables. Rather use `failure_alert`.

Also instead of this pattern, maybe the below reads easier:

```
case val do
  nil -> {:noreply, socket}
  val -> ...
end
```